application/__init__.py

```diff
@@ -1,11 +1,11 @@
-import os
-from flask import Flask, request, jsonify
+from flask import Flask
```

Comment:
Good catch cleaning up these import statements which are now not being used in this file.